src/components/common/navigation/navigation.scss

```diff
@@ -329,4 +331,22 @@
 
 .expandable-active-nav {
     padding: 0px 0 0 12px;
+}
+
+.help__enterprise{
+    display: flex;
```

Comment:
use class which is already available in base.scss & formula.scss

Reply:
Reduced this css class component.